versions.tf

```
@@ -4,5 +4,9 @@ terraform {
   required_providers {
     aws    = ">= 3.0"
     random = ">= 2.1"
+    lacework = {
+      source  = "lacework/lacework"
+      version = "~> 1.0"
```

This has to be at least 1.5 because that's the version we introduced the new resource
```suggestion
      version = "~> 1.5"
```